src/app/Gateways/index.js

```diff
@@ -4,14 +4,12 @@ import FetchCustomerRecord from './FetchCustomerRecord';
 import FetchCustomerDocuments from './FetchCustomerDocuments';
 import SearchCustomers from './SearchCustomers';
 import DeleteCustomerRecord from './DeleteCustomerRecord';
-import FindSnapshots from './FindSnapshots';
```

Comment:
Just checking FindSnapshots was intentionally removed here? Is that not the vulnerabilities thing?

Reply:
Ahh, no ignore me, I see they were just moved elsewhere!